### PR TITLE
Rel Notes doc text for 3.11.285 release

### PR DESCRIPTION
**Pull request by jeana-redhat**

### CODE REVIEW

**Comment by jeana-redhat** [comment]
Preview here: http://file.rdu.redhat.com/~jrouth/20200914/RNs-3_11_285/release_notes/ocp_3_11_release_notes.html#ocp-3-11-285

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -3514,3 +3514,105 @@ Issued: 2020-08-26
...
+balancers do. In this configuration, {product-title} is used through the load
+balancers, but none of the nodes are directly accessible.
+
+A server without a floating IP address is only accessible from the network it is
+in. If any nodes do not have floating IP addresses, then the `openshift-ansible`
+playbooks must be run from inside the node network. A common way to use this
+configuration is to precreate the node network and subnet, create a bastion host
+in it, and run Ansible there.
+
+.Example Steps to Create a Bastion Host
```
Comment (jeana-redhat) [inline release_notes/ocp_3_11_release_notes.adoc:3572]: Note: did not break this up for the purpose of new feature overview

**Review by sheriff-rh — APPROVED** [review]
Huge update, great job!

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -3514,3 +3514,105 @@ Issued: 2020-08-26
...
+[source,terminal]
+----
+$ openstack network create openshift
+$ openstack subnet create --subnet-range 192.168.0.0/24 --dns-nameserver
+ 10.20.30.40 --network openshift openshift
```
Comment (adellape) [inline release_notes/ocp_3_11_release_notes.adoc:3577]: If this is meant to be a single command, should the previous line to get a `\`?

File: release_notes/ocp_3_11_release_notes.adoc [hunk 3]
```diff
@@ -3514,3 +3514,105 @@ Issued: 2020-08-26
...
+$ openstack router create openshift-router
+$ openstack router set --external-gateway public openshift-router
+$ openstack router add subnet openshift-router openshift
+$ openstack server create --wait --image RHEL7 --flavor m1.medium
+--key-name openshift --network openshift bastion
```
Comment (adellape) [inline release_notes/ocp_3_11_release_notes.adoc:3582]: Same here re: potential `\` linebreak.

**Pull request merged**